framework/CustomCommandExecuter.py

```diff
@@ -43,4 +43,4 @@ def execCommand(commandString,self=None,object=None):
     @ In, object, instance, optional, object instance
     @ Out, None
   """
-  execCommandReturn(commandString,self,object)
+  exec(commandString)
```

Comment:
Well, it does have the side effect of putting self and object into the namespace before exec is called.